addons/main/functions/fnc_3den_checkConnection.sqf

```diff
@@ -0,0 +1,9 @@
+params ["_type", "_from", "_to", "_allowedFromClasses", "_allowedToClasses"];
+
+if (!((typeOf _from) in _allowedFromClasses) || !((typeOf _to) in _allowedToClasses) || (_from isEqualTo _to)) then
+{
+    remove3DENConnection [_type, [_from], _to];
+    "forbidden connection removed" call BIS_fnc_3DENNotification;
```

Comment:
Is there a way to explain the issue better?
Because the vanilla connections are symetrical, our approche could be unintuitive

Reply:
In vanilla all connections are allowed. You can sync averything to everything. I never liked that because the user thinks: "If I can do that, then it's valid and will work." But that's not always the case. With these distinct connection types for network and power, It should only be possible to connect things together that are meant to be connected. We could explain more in detail, if I split into separate error messages. But we should still remove connections that are wrong, for example a network connection between desk and chair. I will spit the error messages to be more precise and informative.